### ISSUE TITLE
Improve Docs: a11y: skip nav link

### ISSUE THREAD

**Issue by jbadan** [SAP/fundamental-react#459]
### Description

We should have a skip link like on HIG for keyboard users.

<img width="1151" alt="screen shot 2019-03-05 at 4 27 40 pm" src="https://user-images.githubusercontent.com/29607818/53844650-9ffd4a80-3f63-11e9-8be9-69fd4f06cc2e.png">

### Versions
**fundamental-react:** 0.4.0
**fiori-fundamentals:** N/A

---

_**NOTE:** Where applicable, please include uncropped screen captures._

_**DISCLAIMER:**
After triaging an issue, the fundamental-react team will see if it can be reproduced or confirmed. If more information is needed, the fundamental-react team will contact the author.  Any issues awaiting responses from the author for more than 7 days will be closed.  The author can re-open the issue at a later time if they can present the requested information._
